src/elarm_event.erl

```diff
@@ -72,7 +72,7 @@ unsubscribe(Ref, #evt_state{ subs = Subs } = State)->
 
 -spec new_alarm(alarm(), #evt_state{}) -> {ok, #evt_state{}} | {error, term()}.
 new_alarm(Alarm, #evt_state{ subs = Subs } = State) ->
-    send_events(Alarm, Subs),
+    _ = send_events(Alarm, Subs),
     {ok, State}.
 
 -spec repeat_alarm(alarm(), #evt_state{}) ->
```

Comment:
The spec in line 78 uses a record, please define a type for the record and use that instead.